### DISCUSSION TITLE
Function similar to numpy.clip for symbols

### DISCUSSION THREAD

**Discussion by yuxng** [casadi/casadi#3550] [Q&A] [answered]
Hello!
I am trying to bound a MX matrix with some constant lower bound and upper bound when setting up a function.
A similar function in numpy is numpy.clip https://numpy.org/doc/stable/reference/generated/numpy.clip.html
Any suggestions on how should I implement this in casadi? Thanks!

**Comment by jgillis** [comment, marked as answer]
This looks substitutable by a combination of fmax and fmin..
